tests/common.py

```diff
@@ -901,16 +901,56 @@ def mock_write_data(store, path, data_to_write):
     return await hass.data['system_health']['info'][domain](hass)
 
 
-def mock_integration(hass, module):
+def mock_integration(hass, module, platform_name=None, platform_module=None):
     """Mock an integration."""
     integration = loader.Integration(
         hass, 'homeassisant.components.{}'.format(module.DOMAIN),
         loader.manifest_from_legacy_module(module))
     integration.get_component = lambda: module
 
+    if platform_name is not None:
+        integration.get_platform = \
```

Comment:
This shouldn't be necessary, and can be removed, just like `get_component` stub above. If we stub the cache, it should be good.

Reply:
Maybe, I just realize the platform cache is not set, and the set it in the latest commit